### ISSUE TITLE
Ping spikes while using RFU

### ISSUE THREAD

**Issue by axstin** [axstin/rbxfpsunlocker#275]
Created to prevent duplicate issues.

If you're experiencing problems with network lag or ping spikes while using RFU, please leave a reply answering the following questions:

1) Do the spikes notably affect gameplay (players freezing, rubberbanding, etc.)?
2) Does the issue occur ONLY when RFU is running?
3) If so, do the spikes occur at any FPS cap or only some (e.g. only 144+)?
4) Does the issue occur on any game or only a select few (post a link to the game(s) if this is the case)?

Thanks!

**Comment by TheBananaBoi** [comment]
1. Yes, every time I want to press a button on FE2 It takes about a few seconds to register because of my 2000 ping when I use it.
2. Yes, ONLY when it runs. I've tested it many times. 
3. Not sure since I only used the 240 cap and never other caps. 
4. Usually FE2 For me.  Here's a youtube video on how the ping spikes in FE2: https://youtu.be/EdU9S9dBAeo
And the game link is: https://www.roblox.com/games/738339342/Flood-Escape-2#!/about

**Comment by Teruxity** [comment]
I used to get upwards of 5000 ping when using the FPS Unlocker on Flood Escape 2 (normal ping should be ~100), but today, the ping spikes seem to have disappeared and I have much more stable ping (the same for issue #241). Perhaps the Roblox update might have fixed this for some reason?

**Comment by TheBananaBoi** [comment]
Well, I just tested and it still has the same over 2000ms ping. Sorry for a late response.

**Comment by hirohardtimes** [comment]
Without using fps unlocker i get 170ms to 180ms, when i open fps unlocker ill get around 190ms to 210ms.

**Comment by pogchampgamers** [comment]
I was playing some FPS shooter games without RFU, and it works completely fine.  I have a 144hz monitor, and decided to open up RFU and cap it at 144fps. Players started rubber banding, and everyone froze. I decided to test out every FPS cap (even 30) and it looks like RFU is the problem. I tried other FPS games and even games I've never played before, and then I played them normally without RFU, there was no ping. Seems like RFU just running in the background is causing the game to lag.

**Comment by canekicker** [comment]
1. Once I load up RFU, I experience loaded things disappearing and players rubberbanding and freezing.
2. Yes, my normal ping is like 10-100. With this, my ping just goes up to unexpected amounts, like 2k, 5k, 1k.
3. This amount of ping happens at any fps rate.
4. This happens on every single game that I play. (counter blox, FE2, etc.)

**Comment by XxAbdurrahmanxX10** [comment]
When i open fps unlocker my ping goes to 200 for some reason pls send help

**Comment by zyrnix** [comment]
yeah so, whatever amount of FPS you have, the ping is the same. So lets say you have 75 frames. The ping will also be 75. I tested this out on all options of frames, and it clearly bumps the ping up depending on what FPS you cap it too, even worse if its set at None.

**Comment by LucasMZReal** [comment]
Hm, actually no, I don't see a ping difference at all, I do see, that I sometimes have problems with the game disconnecting, and then not only that, it doesn't show the reconnect prompt, but that seems to be a Roblox Issue and not a RFU issue, but, I never see much people talking about the random disconnects which I personally get. It randomly freezes everything and I'm not sure if thats RFU? I'm gonna to stop using RFU for a bit, just to be sure, but, I think it could be RFU causing disconnecting problems; and then the reconnect prompt is something about roblox i guess;

**Comment by LucasMZReal** [comment]
The disconnecting is like basically the game freezes, everybody is freezed, there's no conecting to the server, but I don't even get a reconnect prompt. I only see client-stuff being loaded, and that's it.

**Comment by s1mple031232132131** [comment]
Do the spikes notably affect gameplay (players freezing, rubberbanding, etc.)? yes players sometimes freezing
Does the issue occur ONLY when RFU is running? yes 
If so, do the spikes occur at any FPS cap or only some (e.g. only 144+)? yes if i set 120 cap then I get a ping that is slightly more than my usual
Does the issue occur on any game or only a select few (post a link to the game(s) if this is the case)?  in all
